### DISCUSSION TITLE
Release 0.1.0

### DISCUSSION THREAD

**Discussion by jbexta** [jbexta/AgentPilot#11] [Announcements]
## 👥 Introducing multi-agent chats with branching history! 🌱

This release brings branching contexts, multi-agent chats and an addition of many more providers through LiteLLM. Combine models from different providers under one context, and configure their interaction with each other in a no-code environment.

### How to migrate your data to 0.1.0
Copy your old database (data.db) to the new application folder before you start the app.

### Release notes:

- Branching chats
- Multi-agent chats
- Cosmetic changes
- New agent settings
- Chat member settings
- Fix open interpreter print bug
- LiteLLM support with 100+ models
- Broken MemGPT until probably a few weeks
- Selected text remains selected while generating
- Other fixes and optimisations
- Some groupchat features aren't working yet

<hr /><em>This discussion was created from the release <a href='https://github.com/jbexta/AgentPilot/releases/tag/'>Release 0.1.0</a>.</em>

**Comment by ishaan-jaff** [comment]
Nice ! thanks for using LiteLLM !!! 

Any feedback on how we can make it better ?

**Reply by jbexta** [reply]
It's working pretty well, thanks for providing litellm! I thought the calls could possibly be object oriented using classes inheriting from a base class, but it makes it easier in one method. All of my GUI is in one file because I find it easier to find my way around

**Reply by jbexta** [reply]
I had some dependency problems migrating to 1.0.0, gonna give that another try sometime soon

**Comment by matari** [comment]
Very exciting project. Voice needs to be the interface.
